app/proguard-debug-rules.pro

```diff
@@ -0,0 +1 @@
+-keepattributes SourceFile,LineNumberTable
```

Comment:
You also want this for the release build, otherwise it will be impossible to make any sense of crash logs.